cinema/serializers.py

```diff
@@ -1 +1,80 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Movie, Actor, Genre, CinemaHall, MovieSession
```

Comment:
Put each from new line

Reply:
According to PEP 8, imports should only be split into new lines if they exceed 79 characters. In this case, it's 71 characters. If you insist, I can reformat it. Should I?

Reply:
If the line suits then keep it